Hanson.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                      = "Hanson"
-    spec.version                   = "1.0"
+    spec.version                   = "1.0.1"
```

Comment:
https://github.com/blendle/Hanson/pull/11

Somehow you're not popping up as a reviewer @jurre.